pandoc.cabal

```diff
@@ -282,6 +283,8 @@ Flag old-locale
 Library
   Build-Depends: base >= 4.7 && < 5,
                  syb >= 0.1 && < 0.8,
+                 split == 0.2.3.2,
```

Comment:
Avoid dependencies on exact versions.
If we really need split, let's depend on as wide a range as possible, at the least `>= 0.2 && < 0.3`.